clorinde/src/codegen/vfs.rs

```diff
@@ -19,7 +19,18 @@ impl Vfs {
     }
 
     /// Add a new file
-    pub fn add(&mut self, path: impl Into<PathBuf>, content: impl Into<String>) {
+    pub fn add(&mut self, path: impl Into<PathBuf>, content: proc_macro2::TokenStream) {
+        let warning = "// This file was generated with `clorinde`. Do not modify.\n\n";
+
+        let syntax_tree = syn::parse2(content).unwrap();
+        let formatted = prettyplease::unparse(&syntax_tree);
```

Comment:
I would prefer using `rustfmt` instead, as prettyplease tends to make some different formatting choices (which in turn make the files being reformatted again whenever I use `cargo fmt` for my project).

Here's what I use for a different codegen lib:

```rust
fn rustfmt(filepaths: Vec<&PathBuf>) -> Result<(), std::io::Error> {
    std::process::Command::new("rustfmt")
        .arg("--edition=2024")
        .args(filepaths)
        .status()?;
    Ok(())
}
```

Reply:
Yeah I was trying to account for people who don't have `rustfmt` installed, but I guess we could just check for that first and use `prettyplease` only as a fallback.

Reply:
That's gotta be uncommon. But sure, fallback sounds nice!